src/utils/response/resourceResponse.ts

```diff
@@ -0,0 +1,26 @@
+import { RawApiResponse } from './rawApiResponse';
+
+export type DeletedEntry = {
+  id: number;
+  deleted_at: string;
+};
+
+export type ResourceResponse<T> = {
```

Comment:
Maybe we can directly use the ApiResponse type here instead of using RawApiResponse or ResoureResponse. They all appear to be the same thing to me

Reply:
But they are not.
* ApiResponse is typed with a T 
* RawApiResponse is not Types
* ResourceResponse is typed with a T but doesn't have all fields from ApiResponse.